package.json

```diff
@@ -123,5 +126,8 @@
     },
     "resolutions": {
         "vite": "5.4.11"
+    },
+    "dependencies": {
+        "big-json": "^3.2.0"
     }
```

Comment:
This belongs to `packages/crawlee/core/package.json`. The top-level `package.json` in this monorepo has only dev-dependencies, not the actual project dependencies